### ISSUE TITLE
Getting segmentation fault

### ISSUE THREAD

**Issue by npm1** [tabatkins/bikeshed#1437]
I just tried installing bikeshed following the steps at https://tabatkins.github.io/bikeshed/#installing and I'm getting a 'Segmentation fault' message every time I try to run bikeshed spec. Is this a known issue, or is there some workaround for this?

**Comment by npm1** [comment]
I notice that the build is having the same problem as me. The latest build fails with segmentation error... it would be good to enforce some checks for commits even if its not full test rebasing.
https://travis-ci.org/tabatkins/bikeshed/builds/494982308?utm_source=github_status&utm_medium=notification
@tabatkins

**Comment by tabatkins** [comment]
Known issue - newest version of LXML (4.3.1) segfaults for some reason. `setup.py` specifies a good max version, but my setup instructions don't currently have you use that file, sorry. :(

You can run `sudo pip install -I lxml==4.3.0`, which'll install the last good version.

This mainly just raises the priority of me swapping in my own HTML parser...

**Comment by npm1** [comment]
BTW If you already have 4.3.1, that might not work and you may want to unistall first:
`sudo pip uninstall lxml`
`sudo pip install -Iv lxml==4.3.0`

**Comment by tabatkins** [comment]
Nah, it definitely installs over the existing 4.3.1; I manually upgraded to 4.3.1, got the crash, then downgraded to 4.3.0 and didn't crash.

**Comment by scoder** [comment]
This is resolved in lxml 4.3.2, freshly released.
